imports/api/Players.js

```diff
@@ -1,6 +1,6 @@
 import { Mongo } from 'meteor/mongo';
 
-//Until some actually halfway programming happens players 
+//Until some actually halfway programming happens players
```

Comment:
Doesn't this PR count as some actually halfway programming?

Reply:
This comment should be removed, is what I'm trying to imply.  Since this pull request involves more advanced player-adding, this complaint comment is obsolete.